platform/android/export/export_plugin.cpp

```diff
@@ -474,7 +474,16 @@ String EditorExportPlatformAndroid::get_valid_basename() const {
 }
 
 String EditorExportPlatformAndroid::get_assets_directory(const Ref<EditorExportPreset> &p_preset, int p_export_format) const {
-	return p_export_format == EXPORT_FORMAT_AAB ? AAB_ASSETS_DIRECTORY : APK_ASSETS_DIRECTORY;
+	String gradle_build_directory = get_gradle_build_directory(p_preset);
+	return gradle_build_directory.path_join(p_export_format == EXPORT_FORMAT_AAB ? AAB_ASSETS_DIRECTORY : APK_ASSETS_DIRECTORY);
+}
+
+String EditorExportPlatformAndroid::get_gradle_build_directory(const Ref<EditorExportPreset> &p_preset) const {
```

Comment:
The method `_is_clean_build_required` checks the `last_gradle_build_time` to determine if a clean build is needed. We should track the `last_gradle_build_dir` and require a clean build as well if the gradle build dir changes.

Reply:
Ah, thanks! Added in my latest push